wait-for-it.sh

```diff
@@ -140,17 +140,23 @@ WAITFORIT_TIMEOUT=${WAITFORIT_TIMEOUT:-15}
 WAITFORIT_STRICT=${WAITFORIT_STRICT:-0}
 WAITFORIT_CHILD=${WAITFORIT_CHILD:-0}
 WAITFORIT_QUIET=${WAITFORIT_QUIET:-0}
-
-# check to see if timeout is from busybox?
+WAITFORIT_ISBUSY=0
+WAITFORIT_BUSYTIMEFLAG=""
 WAITFORIT_TIMEOUT_PATH=$(type -p timeout)
 WAITFORIT_TIMEOUT_PATH=$(realpath $WAITFORIT_TIMEOUT_PATH 2>/dev/null || readlink -f $WAITFORIT_TIMEOUT_PATH)
+
+# check to see if we're using busybox?
 if [[ $WAITFORIT_TIMEOUT_PATH =~ "busybox" ]]; then
-        WAITFORIT_ISBUSY=1
-        WAITFORIT_BUSYTIMEFLAG="-t"
+    WAITFORIT_ISBUSY=1
+fi
 
-else
-        WAITFORIT_ISBUSY=0
-        WAITFORIT_BUSYTIMEFLAG=""
+# see if timeout.c args have been updated in busybox v1.30.0 or newer
```

Comment:
Why not just `timeout --version`?

Reply:
That's possible. It's just that timeout, at least on busybox, doesn't have a --version flag.
```
timeout --version
unrecognized option `--version'
BusyBox v1.22.1 (2015-03-20 11:09:37 GMT) multi-call binary.` 
```

However, it does output the busybox version info in the error, so it could still be used.

It was just a judgment call to use busybox itself to gather the version info since relying on timeout error output felt slightly worse.

Reply:
well, that's a great version. Looks good to me, then!

Reply:
I do not get what is expected by "timeout.c" (".c")
Is it a misprint?